builder.go

```diff
@@ -329,13 +327,13 @@ func (p MailBuilder) SendWithReturnAddress(addr, from string, a smtp.Auth) error
 	for _, a := range p.bcc {
 		recips = append(recips, a.Address)
 	}
-	return smtp.SendMail(addr, a, from, recips, buf.Bytes())
+	return sender.Send(from, recips, buf.Bytes())
 }
 
-// Send encodes the message and sends it via the SMTP server specified by addr. Send uses
-// net/smtp.SendMail, and accepts the same authentication parameters.
-func (p MailBuilder) Send(addr string, a smtp.Auth) error {
-	return p.SendWithReturnAddress(addr, p.from.Address, a)
+// Send encodes the message and sends it via the specified Sender, using the address provided to
+// `From()` as the return address.
```

Comment:
```suggestion
// `From()` as the reverse-path.
```

"Return address" is a bit ambiguous. Is that the `Reply-To` header? Is that the [reverse-path](https://tools.ietf.org/html/rfc5321#section-3.3) (from the `MAIL FROM` command)?

Reply:
Done, thanks